src/toggle-switch/toggle-switch.scss

```diff
@@ -111,11 +118,19 @@
   &:checked {
     background-color: var(--iui-color-background-primary);
     border-color: var(--iui-color-background-primary);
+    @media (forced-colors: active) {
+      background-color: Highlight;
+      border-color: Highlight;
```

Comment:
![image](https://user-images.githubusercontent.com/37936169/163192178-ab675468-c41a-4dbf-8286-1e7fcd42eb3f.png)
The border color doesn't return to the default color if it's been checked previously.

Reply:
I've added lines 98-100, I think they fixed this.

Reply:
They did.